src/Cards/Cards.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react'
 import fetcher from '../API/APIcalls';
 import { Card } from '../Card/Card'
+import { SortMovies } from '../SortMovies/SortMovies'
 import './Cards.scss'
 
 export default class Cards extends Component {
```

Comment:
What was the choice like to make this a class component and fetch all movie when it mounts, rather than fetching on the App component?